sdk/storage/tests/blob.rs

```diff
@@ -327,13 +327,6 @@ async fn copy_blob() {
         .unwrap();
 }
 
-async fn requires_send_future<F, O>(fut: F) -> O
```

Comment:
Why is this checking for `Send`?

Reply:
I believe we want futures which can be sent across thread boundaries. If we're using `tokio` for instance, you want to be able to to use a multi-threaded runtime.

Reply:
IIRC it was a user request.